specification/columns/commitmentdiscountpurchasedquantity.md

```diff
@@ -0,0 +1,38 @@
+# Commitment Discount Purchased Quantity
+
+Commitment Discount Purchased Quantity is the total, positive quantity of [Commitment Discount Units](#commitmentdiscountunit) that is eligible for consumption from a [*commitment discount's*](#glossary:commitment-discount) purchase over a [*row's*](glossary:row) [*charge period*](#glossary:chargeperiod). The CommitmentDiscountPurchaseQuantity column is only applicable to *commitment discounts* and not [*negotiated discounts*](#glossary:negotiated-discount).
+
+The CommitmentDiscountPurchasedQuantity column adheres to the following requirements:
+
+* CommitmentDiscountPurchasedQuantity MUST be present in the billing data when the provider supports *commitment discounts*.
+* CommitmentDiscountPurchasedQuantity MUST be of type Decimal and MUST conform to [Numeric Format](#numericformat) requirements.
+* When [*ChargeCategory*](#chargecategory) is "Purchase", [*ChargeFrequency*](#chargefrequency) is "One-Time", and [*CommitmentDiscountId*](#commitmentdiscountid) is not null, CommitmentDiscountPurchaseQuantity MUST be the total, positive quantity of *CommitmentDiscountUnits*, paid fully or partially upfront, that is eligible for amortization across the *commitment discount's* *term*.
+* When *ChargeCategory* is "Purchase", *ChargeFrequency* is "Recurring", and *CommitmentDiscountId* is not null, CommitmentDiscountPurchasedQuantity MUST be the total, positive quantity of *CommitmentDiscountUnits* that is eligible for amortization for each *charge period* that corresponds with the purchase.
+* CommitmentDiscountPurchasedQuantity MAY be negative, 0, or null in cases where [*ChargeClass*](#chargeclass) is "Correction".
```

Comment:
`0` in case of a correction occurring on a different measure but still wanting to keep the aggregation of this column accurate.
`null` should be removed and this was agreed to be handled in the consistency review.

Reply:
Do you have an example? 

My only concern is that this opens us up to 0 being a compliant value for all refunds.